charts/datafold/values.yaml

```diff
@@ -36,6 +40,15 @@ dfshell:
 
 server:
   install: true
+  command: "server"
```

Comment:
I kept these local to keep the config close to where it's used. If it's not more globally used elsewhere and not used at this level, the preference I chose was to keep it as close as possible to the chart using it.